css/main.scss

```diff
@@ -1796,9 +1795,8 @@ ul {
   }
 }
 fieldset {
-  width: 80%;
-  margin: 0 auto;
   border: none;
+  padding: 0rem;
```

Comment:
is the removal of this 80% fieldset will keep the registration forms intact? Can you check or make this class specific to API explorer? cc @hubert-deriv

Reply:
> is the removal of this 80% fieldset will keep the registration forms intact? Can you check or make this class specific to API explorer? cc @hubert-deriv

You make a good point here. I think this will break the form fields inside the app registration and maybe other places. I suggest reverting this CSS back to what it was and make a new separate specified class where you want to apply these changes.